Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -132,23 +135,44 @@ - (void)resetUpcomingSampleBufferPresentationTimeExpectations;
 
 void VideoMediaSampleRenderer::setTimebase(RetainPtr<CMTimebaseRef>&& timebase)
```

Comment:
Maybe we should add clearTimebase() and expect setTimebase to only be called when m_timebase is null.

Reply:
yes, good point that may make it more clear, it could be private too so it's even more clear that you can only set the timebase once and avoid making usage error